homeassistant/components/notion/sensor.py

```diff
@@ -1,16 +1,20 @@
 """Support for Notion sensors."""
-from homeassistant.components.sensor import SensorEntity
+from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
```

Comment:
```suggestion
from __future__ import annotations

from homeassistant.components.sensor import SensorEntity, SensorEntityDescription
```

Reply:
I don't think this is necessary as of 5e65b27 (since the type checker can properly infer everything), yes?

Reply:
Yes, only necessary with `tuple[...]` or `dict[...]` annotation as they normally would require Python 3.9